### PR TITLE
Bump ORCA to v3.101.0

### PR DESCRIPTION
**Pull request by sambitesh**
## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: config/orca.m4 [hunk 1]
```diff
@@ -40,7 +40,7 @@ AC_RUN_IFELSE([AC_LANG_PROGRAM([[
 #include <string.h>
 ]],
 [
-return strncmp("3.100.", GPORCA_VERSION_STRING, 6);
+return strncmp("3.101.", GPORCA_VERSION_STRING, 6);
 ])],
 [AC_MSG_RESULT([[ok]])],
 [AC_MSG_ERROR([Your ORCA version is expected to be 3.100.XXX])]
```
Comment (chrishajas) [inline config/orca.m4:46]: update this line

File: configure [hunk 2]
```diff
@@ -12523,7 +12523,7 @@ int
 main ()
 {
 
-return strncmp("3.100.", GPORCA_VERSION_STRING, 6);
+return strncmp("3.101.", GPORCA_VERSION_STRING, 6);
```
Comment (chrishajas) [inline configure:12526]: also configure.in

Comment (sambitesh) [reply configure:12526]: Thanks

File: config/orca.m4 (hunk 1 above)
Comment (sambitesh) [reply config/orca.m4]: Done

**Pull request merged**